spec/index.html

```diff
@@ -1623,9 +1630,12 @@ <h3>The <code>rdf:JSON</code> Datatype</h3>
               Such values cannot be represented as JSON Numbers, limiting the ability to serialize such values back to JSON.</div>
           </li>
           <li>A <a href="RFC8259#section-7">JSON String</a> is mapped to
-            a <a>string</a>.</li>
-          <li>A <a href="RFC8259#section-3">JSON literal name</a> is mapped to
-            <a data-cite="INFRA#boolean">`true`, `false`</a>, or <a data-cite="INFRA#nulls">null</a>.</li>
+            a <a>string</a> after converting any escape sequences to the associated
+            <a data-cite="I18N-GLOSSARY#dfn-code-point">Unicode code point</a>.</li>
+          <li>A <a href="RFC8259#section-3">JSON literal name</a> maps
+            `true` to <a data-cite="INFRA#boolean">`true`</a>,
+            `false` to <a data-cite="INFRA#boolean">`false`</a>, and
+            `null` to <a data-cite="INFRA#nulls">`null`</a>.</li>
```

Comment:
I wonder whether there should be some difference in the visible target text? Something like this, perhaps?
```suggestion
            `true` to <a data-cite="INFRA#boolean">Infra boolean `true`</a>,
            `false` to <a data-cite="INFRA#boolean">Infra boolean `false`</a>,
            and `null` to <a data-cite="INFRA#nulls">Infra `null`</a>.</li>
```

Reply:
Maybe something like the following:

A JSON literal name maps the values `true`, `false, and `null` to <a data-cite="INFRA#boolean">`true`</a>, <a data-cite="INFRA#boolean">`false`</a>, and <a data-cite="INFRA#nulls">`null`</a> from [[INFRA]], respectively.

Reply:
Does this work?
```suggestion
            JSON `true`, `false`, and `null` values
            to [[INFRA]] `true`, `false`, and `null` values,
            respectively.</li>
```

Reply:
That's what I was thinking, other than having the `true`, `false`, and `null` values from INFRA cited as is the current practice.

Reply:
```suggestion
            JSON `true`, `false`, and `null` values
            to [[INFRA]] <a data-cite="INFRA#boolean">`true`</a>,
            `false` to <a data-cite="INFRA#boolean">`false`</a>, and
            `null` to <a data-cite="INFRA#nulls">`null`</a> values, respectively.</li>
```